app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: str) -> None | str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            Cafe(cafe).visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+    if masks_to_buy > 0:
+        return f"Friends should buy {masks_to_buy} masks"
+    return f"Friends can go to {cafe.name}"
```

Comment:
On this line, you're trying to access the 'name' attribute of 'cafe', which is a string, not an object. This will cause an error. You probably meant to refer to the 'cafe' object created inside the 'try' block, but it's not accessible here because it's in a different scope. Consider revising your code structure.